ValueSet/ValueSet-DM-prescription-type.xml

```diff
@@ -2,14 +2,19 @@
 <ValueSet xmlns="http://hl7.org/fhir">
     <id value="043b4a0c-1677-4d95-b804-9880ef818e27"/>
     <url value="https://fhir.nhs.uk/ValueSet/DM-prescription-type"/>
-    <version value="1.0.4"/>
+    <version value="1.0.5"/>
     <name value="DMPrescriptionType"/>
     <status value="draft"/>
     <description value="Subset on UK Core Prescription Therapy Type which is permitted for EPS use."/>
     <compose>
         <include>
             <system value="https://fhir.nhs.uk/CodeSystem/prescription-type"/>
-            <version value="1.1.4"/>
+            <version value="1.1.5"/>
+            <filter>
```

Comment:
Just to confirm my reading / understanding of this filter - this will filter the valueset to the codesystem entries which DO NOT have a deprectaed property, regardless of its value?

Reply:
Correct, we are only adding the deprecated property to codes which have been deprecated, so an absence of the property implies the concept is not deprecated (I couldn't find a != operator in the FHIR op guidance, otherwise I would have changed the expression to "deprecated != true")